pkg/unstructured/xml_config.go

```diff
@@ -143,9 +144,12 @@ func (x *xmlConfig) Marshal() (string, error) {
 
 func (x *xmlConfig) Unmarshal(str string) error {
 	m, err := mxjv2.NewMapXml([]byte(str), true)
-	if err != nil {
+	if err != nil && err != io.EOF {
 		return err
 	}
+	if err == io.EOF {
+		m = mxjv2.New()
```

Comment:
When the content is invalid XML, `mxj` returns an `io.EOF` error as well. So if `str` is not empty, the error should be reported instead of silently ignored.

![image](https://github.com/user-attachments/assets/7b4bde34-ac9a-4c7e-bc04-1bf54e9d8451)